### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by alex1779**
In this project, I will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.

- Adding index html, css style sheet, js file, webpack configuration, dist files.
- Adding license file.
- Update readme file.
- Adding linters files.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+  <div class="icon">&#8942</div>
+</li>
+`;
+  const inputText = document.querySelector('#inputNewTask');
+  inputText.value = '';
+}
+
+window.onload = () => {
+  // const button = document.querySelector('.clearAll');
+  // button.onclick = function name() { sayHi(); };
```
Comment (brytebee) [inline src/index.js]: - [ ] **Optional:** Kindly remove unused code. This makes your code clean and professional.

File: src/index.js [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
-
-  element.appendChild(btn);
-
-
-   return element;
- }
-
- document.body.appendChild(component());
\ No newline at end of file
+const tasksArray = [];
```
Comment (brytebee) [inline src/index.js]: - [ ] **Required:** Kindly ensure that this is populated with objects that have the following or similar attributes:

- Complated,
- Description,
- Index

Which are rendered dynamically to the page.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @alex1779👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @alex1779, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**